isis/src/base/objs/SpiceRotation/SpiceRotation.h

```diff
@@ -187,31 +187,37 @@ namespace Isis {
    *                           any NAIF errors were signaled. References #2248.
-   *   @history 2015-08-05 Debbie A. Cook - Programmer notes - Modified LoadCache, 
+   *   @history 2015-08-05 Debbie A. Cook - Programmer notes - Modified LoadCache,
    *                           and ComputeAv.
-   *                           Added new methods 
+   *                           Added new methods
    *                           loadPCFromSpice, loadPCFromTable, toJ2000Partial, poleRaCoefs,
-   *                           poleDecCoefs, pmCoefs, poleRaNutPrecCoefs, poleDecNutPrecCoefs, 
+   *                           poleDecCoefs, pmCoefs, poleRaNutPrecCoefs, poleDecNutPrecCoefs,
    *                           pmNutPrecCoefs, sysNutPrecConstants, sysNutPrecCoefs,
    *                           usePckPolynomial, setPckPolynomial(raCoef, decCoef, pmCoef),
    *                           getPckPolynomial, setEphemerisTimePckPolyFunction, getFrameType
-   *                           and members m_frameType, m_tOrientationAvailable, 
+   *                           and members m_frameType, m_tOrientationAvailable,
    *                           m_raPole, m_decPole, m_pm, m_raNutPrec, m_decNutPrec, m_pmNutPrec,
-   *                           m_sysNutPrec0, m_sysNutPrec1, m_dscale, m_Tscale to support request for 
+   *                           m_sysNutPrec0, m_sysNutPrec1, m_dscale, m_Tscale to support request for
    *                           solving for target body parameters.
-   *                           Also added a new enumerated value for Source, PckPolyFunction, 
+   *                           Also added a new enumerated value for Source, PckPolyFunction,
    *                           and PartialType, WRT_RotationRate.
    *   @history 2016-02-15 Debbie A. Cook - Programmer notes - Added private method
-   *                           setFrameType to set the frame type.  It also loads the planetary 
-   *                           constants for a PCK type. 
+   *                           setFrameType to set the frame type.  It also loads the planetary
+   *                           constants for a PCK type.
    *   @history 2016-06-28 Ian Humphrey - Updated documentation and coding standards. Added new
    *                           tests to unit test. Fixes #3972.
    *   @history 2017-12-13 Ken Edmundson - Added "case DYN:" to methods ToReferencePartial and toJ2000Partial. Fixes #5251.
    *                           This problem was found when trying to bundle M3 images that had been spiceinited with nadir
    *                           pointing. The nadir frame is defined as a Dynamic Frame by Naif.
+   *   @history 2018-04-21 Jesse Mapel - Modified frametype resolution to check if a body centered
+   *                           frame uses a CK or PCK definition. This only occurs for bodies
+   *                           where a pck cannot accurately define for the duration of a mission.
```

Comment:
small typo: change "...where a pck cannot accurately define for..." to something like "...cannot be accurately defined..."  or "...cannot accurately define _it_ for..."